js/latoken.js

```diff
@@ -275,7 +308,15 @@ module.exports = class latoken extends Exchange {
 
     async fetchBalance (currency = undefined, params = {}) {
         await this.loadMarkets ();
-        const response = await this.privateGetAccountBalances (params);
+        let response;
+        if (currency === undefined) {
+            response = await this.privateGetAccountBalances (params);
```

Comment:
The same is true here, we need to remove the `currency` argument and always call the "all balances".